src/main/scala/beam/agentsim/agents/ridehail/repositioningmanager/DemandFollowingRepositioningManager.scala

```diff
@@ -130,17 +130,23 @@ class DemandFollowingRepositioningManager(val beamServices: BeamServices, val ri
   private def findWhereToReposition(tick: Int, vehicleLocation: Coord, vehicleId: Id[Vehicle]): Option[Coord] = {
     val currentHour = tick / 3600
     val nextHour = currentHour + 1
+    val percentageOfClosestClusters =
+      beamServices.beamConfig.beam.agentsim.agents.rideHail.repositioningManager.demandFollowingRepositioningManager.PercentageOfClosestClustersToConsider
+
     hourToClusters.lift(nextHour).map { clusters =>
-      // We get top 5 closest clusters and randomly pick one of them.
+      val N = (clusters.length * percentageOfClosestClusters).toInt
```

Comment:
Please round to nearest whole number, "toInt" is a "floor" operation and could confuse users